test/release/examples/benchmarks/hpcc/README

```diff
@@ -13,7 +13,7 @@ approximately as follows:
 * The STREAM and RA benchmarks are scalable but have outstanding
   performance and elegance improvements that we plan to address.
 
-* The FFT, PTRANS, and HPL benchmarks run on distributed memory, but
+* The FFT, and PTRANS benchmarks run on distributed memory, but
```

Comment:
I think there's going to be a merge conflict with the FFT PR, and that you'll probably want to fix this sentence to refer to a single benchmark grammatically.

Reply:
Yeah, it's what I get for making a fresh branch instead of shoving them into one.  Ah well